gcsio/src/main/java/com/google/cloud/hadoop/gcsio/DeleteFolderOperation.java

```diff
@@ -163,7 +163,7 @@ private void addFolderResourceInBlockingQueue(FolderInfo folderResource) {
    *
    * @param folderResource of the folder that is now deleted
    */
-  private void successfullDeletionOfFolderResource(FolderInfo folderResource) {
+  void successfullDeletionOfFolderResource(FolderInfo folderResource) {
```

Comment:
If we are increasing the visibility for testing, why not make this protected instead of private?

Reply:
Changed access modifiers